app/build.gradle

```diff
@@ -6,10 +6,10 @@ apply plugin: 'kotlin-kapt'
 
 android {
   compileSdkVersion 33
-  buildToolsVersion "29.0.2"
+  buildToolsVersion "30.0.2"
```

Comment:
This specific version was chosen because this was the minimum required version for our selected compose version. There might be other issues trying to further upgrade the build tools, not sure about it at the moment.